docs/source/examples/pt_zne.md

```diff
@@ -0,0 +1,386 @@
+```
+
+**Incoherent noise** is a process that results in the quantum system entangling with its environment i.e. this type of noise is irreversible. The system and the environment end up in a mixed state. It scales linearly in the small error limit. 
+The noise channel can be described using Pauli operators which makes it easy to analyze and simulate. Worst case error rate is directly proportional to the **average gate infidelity**. 
+
+For example, a depolarizing noise channel is a stochastic noise channel where a noiseless process is probabilistically mixed with orthogonal errors. If $\rho$ is a single qubit state, $p$ is the probabilistic error rate and $\mathcal{E}(\rho)$ is the noise channel:
+
+```{math}
+\mathcal{E}(\rho) = (1-p) \rho + p \frac{I}{2}
+```
+$\frac{I}{2}$ is the maximally mixed state which can be described using Paulis. 
+
+```{math}
+\frac{I}{2} = \frac{1}{4} (\rho + X \rho X + Y \rho Y + Z \rho Z)
+```
+
+Thus, the depolarizing channel can be redescribed using Paulis as shown below. 
+
+```{math}
+\mathcal{E}(\rho) = (1-\frac{3p}{4}) \rho + \frac{p}{4} (X \rho X + Y \rho Y + Z \rho Z)
+
+```
+
+### Pauli Transfer Matrix
+
+Let $\Lambda(\rho)$ be a $n$-qubit noise channel with $K_i$ being the corresponding Kraus operators.
+
+```{math}
+\Lambda(\rho) = \sum_{i=1} K_i \rho {K_i}^\dagger
+```
+If $P_i$ and $P_j$ are lexicographically ordered $n$-qubit Paulis $\forall P_i, P_j \in \{I, X, Y, Z \}^{\otimes n}$, the following expression defines the entries of a Pauli Transfer Matrix (PTM). Here, 
+$i$ defines the rows while $j$ defines the columns of the PTM. 
+
+
+```{math}
+(R_{\Lambda})_{ij} = \frac{1}{2^n} \text{Tr} \{ P_i \Lambda(P_j)\}
+```
+
+All entries of the PTM are real and in the interval $[-1, 1]$. A PTM allows us to distinguish between the two types of noise. The off-diagonal terms of the PTM are due to the effect of coherent noise while the diagonal terms are due to incoherent noise. To find the PTM of an entire circuit, we only need to take the product of the PTM of each layer in the circuit. Due to this, it is straightforward to see how coherent noise carries across different layers in the circuit and how incoherent errors are easier to deal with in the small error limit. The latter is due to only focusing on the diagonal terms of the PTM for incoherent noise such that the product of two or more diagonal matrices is also a diagonal matrix.
+
+The known fault tolerant thresholds for stochastic noise are higher than coherent noise which makes the former a 'preferable' type of noise compared to the latter. If we want to not deal with coherent noise, Pauli twirling can be used to tailor coherent noise to incoherent noise. Same as the expression above, when a coherent noise channel is Pauli twirled, the noise channel can be described using Paulis after averaging over multiple Pauli twirled circuits. Refer to the [Pauli Twirling user guide](../guide/pt.md) for additional information. 
```

Comment:
> If we want to not deal with coherent noise, ...

To avoid dealing with coherent noise, Pauli twirling can be applied to convert coherent noise into incoherent noise.

> Same as the expression above, ...

Which expression above are you referring to?